### ISSUE TITLE
Friction is not being set from task logic

### ISSUE THREAD

**Issue by bruno-f-cruz** [AllenNeuralDynamics/Aind.Behavior.VrForaging#303]
I suspect a bug with the Interpolator as there are several 0s logged as harp messages. This is weird as DistinctUntilChanged is applied prior to the interpolator.

**Comment by bruno-f-cruz** [comment]
Does not appear to be a bug with the interpolator implementation either. A test (F(x), x) shows that everything is working as expected:
```
(0, 0)
(6553, 0.1)
(13107, 0.2)
(19660, 0.3)
(26214, 0.4)
(32767, 0.5)
(39321, 0.6)
(45874, 0.7)
(52428, 0.8)
(58981, 0.9)
```

**Comment by bruno-f-cruz** [comment]
@tiffanyona can you add the task logic json and commit necessary to reproduce this bug?

**Comment by tiffanyona** [comment]
Yes! 
Commit: 9c6cc34
Task logic:
[test_friction.json](https://github.com/user-attachments/files/17080204/test_friction.json)

I used rig 5A for these tests, in case you want to do the same

**Comment by bruno-f-cruz** [comment]
Sorry I meant the rig schema with the calibration.

**Comment by bruno-f-cruz** [comment]
Turns out the 0.4.x version had a bug in the interpolator that has been fixed in the proposed 0.5 release. I have patched it and made a new 0.4.3 release with this fix!
